x/incentives/keeper/gauge.go

```diff
@@ -127,9 +127,42 @@ func (k Keeper) CreateGauge(ctx sdk.Context, isPerpetual bool, owner sdk.AccAddr
 		return 0, types.ErrZeroNumEpochsPaidOver
 	}
 
+	// If the gauge has no lock, then we assume it is a concentrated pool and ensure
+	// the gauge "lock" duration is an authorized uptime.
+	isConcentratedPoolGauge := distrTo.LockQueryType == lockuptypes.NoLock
+
+	// If the gauge has an internal gauge denom, it is an internal gauge
+	// and should be run through different validation logic (see below).
+	//
+	// Two important reminders/assumptions:
+	// 1. `NoLock` gauges are required to have empty denoms in `ValidateBasic`, so this
+	// check cannot be controlled by user input.
+	// 2. The safety of this leans on the special-casing of internal gauge logic during
+	// distributions, which should be using the internal incentive duration gov param instead of the duration value.
+	isInternalConcentratedPoolGauge := distrTo.Denom == types.NoLockInternalGaugeDenom(poolId)
```

Comment:
To make this stricter. Ran the tests - they still pass
```suggestion
	isInternalConcentratedPoolGauge := isConcentratedPoolGauge && distrTo.Denom == types.NoLockInternalGaugeDenom(poolId)
```

Reply:
34bba93dc0d82eb528c4a9c123a27abc6d912dbb